db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170319214240) do
+ActiveRecord::Schema.define(version: 20170613071826) do
```

Comment:
Migration files can be merged all into a final one instead of having many files.